hydrolib/core/io/mdu/models.py

```diff
@@ -30,7 +30,7 @@ class General(INIGeneral):
     version: str = Field("1.2.94.66079M", alias="version")
     filetype: Literal["modelDef"] = Field("modelDef", alias="fileType")
     fileversion: str = Field("1.09", alias="fileVersion")
-    autostart: bool = Field(False, alias="autoStart")
+    autostart: int = Field(0, alias="autoStart")
```

Comment:
We also like to use enums for limited options such as in this case. 
This comes with the advantage that we can also document the several options. For example that `0` is `no`, `1` = `autostart` and `2` = `autostartstop`. 

I _think_ (not sure) that we get also "free" validation on the values, for example that 3 or -1 is not valid.


```
class AutoStartOption(IntEnum):
    """
    Enum class containing the valid values for the AutoStart
    attribute in the [General][hydrolib.core.io.mdu.models.General] class.
    """
    no=0
    autostart=1
    autostartstop=2
```


```
autostart: AutoStartOption = Field(AutoStartOption.no, alias="autoStart")
```